node_modules/oae-principals/tests/test-groups.js

```diff
@@ -101,7 +101,7 @@ describe('Groups', function() {
                 assert.ok(groupObj.lastModified);
                 assert.ok(groupObj.lastModified >= before);
                 assert.ok(groupObj.lastModified <= Date.now());
-                assert.equal(groupObj.lastModified, groupObj.created);
+                assert.equal(new Date(parseInt(groupObj.lastModified)).toUTCString(), groupObj.created);
```

Comment:
Will this continue to pass since we don't use `lastModified` for `created` above anymore?